src/main/resources/db/migration/V1__Init_tables.sql

```diff
@@ -0,0 +1,141 @@
+DROP TABLE IF EXISTS cc_employee CASCADE;
+DROP TYPE IF EXISTS cc_employee_status;
+DROP TYPE  IF EXISTS cc_employee_role;
+
+CREATE TYPE cc_employee_status as enum ('INACTIVE','ACTIVE','ARCHIVED');
+CREATE TYPE cc_employee_role as enum ('STAFF','ADMIN');
```

Comment:
Do we really need to create these enum types at DB level? Or we can just control them at cod level instead?